src/components/homepage/JoinCTA.astro

```diff
@@ -0,0 +1,23 @@
+<section class="w-full flex-col justify-start items-center bg-gray-700 text-white py-8">
+  <div class="text-center mb-12">
+    <h2 class="text-3xl md:text-5xl font-bold my-4 mx-4 sm:mx-8">
+      Join Our Open Source Movement
+      <!-- Call To Action! -->
+    </h2>
+    <p class="text-lg md:text-xl mb-10 mx-4 sm:mx-8">
+      Contribute your skills and knowledge to help build a vibrant alumni portal for NCIT. Your expertise can make a difference!
+      
+    </p>
+    <div class="flex flex-col sm:flex-row justify-center items-center space-y-4 sm:space-y-0 sm:space-x-4">
+      <a href="https://github.com/noskofficial" target="_blank" class="border border-green-500 hover:bg-green-500 text-white py-3 px-6 transition-all duration-300">
+        Contribute Now
+      </a>
+      <a href="/learn_more" class="border border-yellow-500 hover:bg-yellow-500 text-white py-3 px-6 transition-all duration-300">
+        Learn More
+      </a>
+    </div>
+    <p class="text-lg md:text-xl pt-8 mx-4 sm:mx-8">
```

Comment:
Also this <p> ... </p> can be removed. What say @axyut @mukezhz ?

Reply:
sure, it does look out of place but `Call for action` was a good term so I added it.